addons/fleet/wizard/fleet_vehicle_send_mail.py

```diff
@@ -32,14 +36,21 @@ def action_send(self):
                 }
             }
 
+        if self.template_id:
+            subjects = self._render_field('subject', res_ids=self.vehicle_ids.ids)
+            bodys = self._render_field('body', res_ids=self.vehicle_ids.ids)
```

Comment:
Tiny nit-picks:
1. The plural of body is bodies, not bodys. (applies also on line 44 and 49) 
2. I think that including the parameter name for the field, like you do for res_ids, can make it a bit clearer.

```suggestion
            bodies = self._render_field(field='body', res_ids=self.vehicle_ids.ids)
```

Reply:
Indeed, could you adapt please ? 

Thanks.

Reply:
oups i didn't see your comment :grimacing: